pkg/destroy/openstack/openstack.go

```diff
@@ -1822,3 +1832,69 @@ func validateCloud(opts *clientconfig.ClientOpts, logger logrus.FieldLogger) err
 
 	return networkextensions.Validate(availableExtensions)
 }
+
+func cleanVIPsPorts(opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger) (bool, error) {
+	logger.Debug("Cleaning provided Ports for API and Ingress VIPs")
+	defer logger.Debugf("Exiting clean of provided Ports for API and Ingress VIPs")
+	conn, err := clientconfig.NewServiceClient("network", opts)
+	if err != nil {
+		logger.Error(err)
+		return false, nil
+	}
+
+	tag := filter["openshiftClusterID"] + openstackdefaults.DualStackVIPsPortTag
+	PortlistOpts := ports.ListOpts{
+		TagsAny: tag,
+	}
+	allPages, err := ports.List(conn, PortlistOpts).AllPages()
+	if err != nil {
+		logger.Error(err)
+		return false, nil
+	}
+
+	allPorts, err := ports.ExtractPorts(allPages)
+	if err != nil {
+		logger.Error(err)
+		return false, nil
+	}
+
+	numberToClean := len(allPorts)
+	numberCleaned := 0
+
+	// Updating user provided API and Ingress Ports
+	if len(allPorts) > 0 {
+		fipByPort, err := getFIPsByPort(conn, logger)
+		if err != nil {
+			logger.Error(err)
+			return false, nil
+		}
+		for _, port := range allPorts {
+			var sgs []string = nil
+			logger.Debugf("Updating security groups for Port: %q", port.ID)
+			_, err := ports.Update(conn, port.ID, ports.UpdateOpts{SecurityGroups: &sgs}).Extract()
```

Comment:
Not really a big deal, but it might be that user will have some SGs attached to the port. To me just a comment explaining this simplification would be sufficient here.

Reply:
Good point, I fixed it to make sure of installer sgs are cleaned